build/shade/_k-standard-goals.shade

```diff
@@ -97,8 +97,35 @@ default PACKAGELIST_JSON_FILENAME = 'NuGetPackageVerifier.json'
 
 #build-compile target='compile' if='Directory.Exists("src")'
   @{
+    
+    Directory.CreateDirectory(TARGET_DIR);
+    
+    string commitHash = null;
+    if (AddAssemblyInfo)
+    {
+        var commitHashFile = Path.Combine(TARGET_DIR, "commit");
+        GitCommand("rev-parse HEAD >> " + commitHashFile);
+        commitHash = File.ReadAllLines(commitHashFile)[0];
+    }
+  
     var projectFiles = Files.Include("src/*/project.json").ToList();
-    projectFiles.ForEach(projectFile => DotnetPack(projectFile, BUILD_DIR, Configuration));
+    projectFiles.ForEach(projectFile => 
+    {
+        if (AddAssemblyInfo)
+        {
+            var projectFolder = Path.GetDirectoryName(projectFile);
+            var assemblyInfoFile = Path.Combine(projectFolder, "Properties", "AssemblyInfo.cs");
+            if (File.Exists(assemblyInfoFile))
```

Comment:
So keeping this as an "opt-in" with the environment variable is better? (and not changing the local builds)